pkg/cvo/status.go

```diff
@@ -27,8 +27,7 @@ import (
 
 const (
 	// ClusterStatusFailing is set on the ClusterVersion status when a cluster
-	// cannot reach the desired state. It is considered more serious than Degraded
-	// and indicates the cluster is not healthy.
+	// cannot reach the desired state. It indicates the cluster is not healthy.
```

Comment:
Why can not we say that it probably means one or more operators are in degraded state, rather than saying not healthy?

Reply:
because there could be other reasons besides unavailable/degraded operators to be `Failing=True`.  Although without #867 in place, it's hard to get Telemetry stats on how frequent the various modes are.

Reply:
Though this is not customer facing documentation still makes me nervous about telling cluster is not healthy but we do not think this is more serious degraded condition of operators.

Reply:
`Failing` may be more serious than `Degraded` (e.g. it may be because a ClusterOperator is `Available=False`).  `Failing` may be less serious than `Degraded` (e.g. we may be having trouble rolling out a peripheral alert rule).  I'm not saying `Failing` is not  serious.  I'm just dropping the apples-to-oranges `Degraded` comparison.